src/service/standard-ampdoc-services.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {adoptServiceForEmbedDoc} from './service';
+import {installActionServiceForDoc} from './service/action-impl';
+import {installCidService} from './service/cid-impl';
+import {installDocumentInfoServiceForDoc} from './service/document-info-impl';
+import {installGlobalNavigationHandlerForDoc} from './service/navigation';
+import {installGlobalSubmitListenerForDoc} from './document-submit';
+import {installHiddenObserverForDoc} from './service/hidden-observer-impl';
+import {installHistoryServiceForDoc} from './service/history-impl';
+import {installResourcesServiceForDoc} from './service/resources-impl';
+import {installStandardActionsForDoc} from './service/standard-actions-impl';
+import {installStorageServiceForDoc} from './service/storage-impl';
+import {installUrlForDoc} from './service/url-impl';
+import {installUrlReplacementsServiceForDoc} from './service/url-replacements-impl';
+import {installViewerServiceForDoc} from './service/viewer-impl';
+import {installViewportServiceForDoc} from './service/viewport/viewport-impl';
+
+
+/**
+ * Install ampdoc-level services.
+ * @param {!./service/ampdoc-impl.AmpDoc} ampdoc
+ * @param {!Object<string, string>=} opt_initParams
+ */
+export function install(ampdoc, opt_initParams) {
+  // Order is important!
+
+  // QQQQQQ: all service passthrough to embed are bad here. some are near-critical.
+  // remove them asap.
```

Comment:
Isn't status quo that all services pass-through/inherit except the following?
```
url, actions, standard-actions, navigation, timer, bind
```
Seems like we should keep that behavior at least for MVP migration.

Reply:
It's actually not, no. You're right that we may hit some compatibility blocks and for that reason I will have passthrough under experiment. But the final state: no passthrough. In very rare cases where passthrough is needed we just need to do it explicitly.

Reply:
"No pass-through" sounds kind of heavy. Are you thinking this is better for security?

 Also, does it prevent lightweight, "root" use cases?

Reply:
I think we definitely need "open" and "close" modes. And I'm pretty sure they will be easier to accomplish for us. But I want to focus on "close" first since that covers our most current cases. "Open" roots would take some more discussion to figure out.